src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Table.cs

```diff
@@ -74,7 +74,7 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
                 }
                 else
                 {
-                    if (DType.TryUnionWithCoerce(rowType, argType, context.Features.PowerFxV1CompatibilityRules, out var newType, out bool coercionNeeded))
+                    if (DType.TryUnionWithCoerce(rowType, argType, context.Features.PowerFxV1CompatibilityRules, coerceToLeftTypeOnly: true, out var newType, out bool coercionNeeded))
```

Comment:
>coerceToLeftTypeOnly: true, [](http://example.com/codeflow?start=113&length=27)

Should this be only under Fx compat flag?

Reply:
Updated to only do that under V1 compat rules